projects/opencensus-cpp/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://opencensus.io/"
+language: c++
+primary_contact: "opentelemetry-dev+opencensus-fuzz@google.com"
```

Comment:
Unfortunately the "+" breaks the auth for ClusterFuzz here, even if this was an email associated with an individual google account. In this case though this looks like a mailing list so it's unlikely to work anyway.

@jsuereth could you please suggest other individual emails to add here? Filtering these emails should be easy.

Reply:
You can use joshuasuereth@google.com for now then.